dev/how-to/maintain/backup-and-restore.md

```diff
@@ -53,15 +53,15 @@ To quickly backup and restore data (especially large amounts of data), refer to
 
 - The total size of the exported files is 214G. A single table has 8 columns and 2 billion rows.
 - The cluster topology:
-  - 12 TiKV instances: 4 nodes, 3 TiKV instances per node
-  - 4 TiDB instances
-  - 3 PD instances
+    - 12 TiKV instances: 4 nodes, 3 TiKV instances per node
```

Comment:
To make one list nested in another, indent two spaces from the first letter of the list to be nested. Therefore, the format is here correct and needs no more spaces. FYR: https://guides.github.com/features/mastering-markdown/#syntax